docs/getting-started/index.md

```diff
@@ -25,7 +25,7 @@ We recommended installing Ruby locally for development because:
 
 ## Development with Ruby
 
-#### Windows Instructions
+<h3 data-toc-skip class="h4">Windows Instructions</h3>
```

Comment:
```suggestion
### Windows Instructions
{:toc-skip}
```
See comment on community/index.md regarding use of Kramdown attribute list definitions to avoid inline HTML.